sympy/diffgeom/diffgeom.py

```diff
@@ -1856,6 +1855,8 @@ def intcurve_diffequ(vector_field, param, start_point, coord_sys=None):
     intcurve_series
 
     """
+    from sympy.simplify import simplify
```

Comment:
Can the import here just go at the end of the file?

Importing at the end of the file is better than importing in a function because it means that the import only happens once.

Reply:
Yes, it was possible here. Tried the other files as well, but not possible there.